addons/github/serializer.py

```diff
@@ -34,7 +34,7 @@ def addon_serialized_urls(self):
                                 service_name='github'),
             'importAuth': node.api_url_for('github_import_auth'),
             'files': node.web_url_for('collect_file_trees'),
-            'folders': node.api_url_for('github_root_folder'),
```

Comment:
Do we still need to expose this `github_root_folder` url?

Reply:
Yes we still need it but not here, other providers like dropbox, box etc the folder structure begins at the root folder level so showing the contents of the root folder is the same as listing all the folders, but for Github we start at the repo level, so these need to be used separately. github_folder_list actually list all repos and github_root_folder is used once the repo is connected to list all the folders hierarchically